sympy/matrices/reductions.py

```diff
@@ -288,6 +288,22 @@ def _rref(M, iszerofunc=_iszero, simplify=False, pivots=True,
     >>> rref_pivots
     (0, 1)
 
+    iszerofunc can correct rounding errors in matrices with float values.
+    In the following example, calling rref leads to floating point errors,
+    incorrectly row reducing the matrix. iszerofunc can set sufficiently
+    small numbers to zero and avoid this error.
```

Comment:
There should be a blank line between the text and the example.

Reply:
Alright, I just added the line.

Reply:
Consider using formatting to make the text render more clearly. See https://docs.sympy.org/dev/guides/contributing/documentation-style-guide.html

Reply:
I believe I made the correct formatting changes.